### PR TITLE
connected postgrel db

### PR DESCRIPTION
**Pull request by efosdan**

### CODE REVIEW

**Review by courage173 — CHANGES_REQUESTED** [review]
this looks good. but i checked your initial commit you should set your port from .env and you name your file server.js not index.js

const port = process.env.PORT || 3000

**Comment by efosdan** [comment]
so i should rename the index.js or seperate the server from the app?

File: src/server.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
 const app = express();
-const port = 3000
+const port = process.env.PORT || 3000
 
 app.listen(port, ()=>{
     console.log(`Team server listening at port ${port}!`)
-})
\ No newline at end of file
+})
+client.connect();
```
Comment (courage173) [inline src/server.js]: call the client.connect inside the call back where you listen to the port. incase there are any errors you don't connect. also add a log if the database has connected

Comment (efosdan) [reply src/server.js]: there was suddenly an error in the node modules and serrver could not run. it says could not find index.js until i rename server.js to index.js and deleted the node modules and package-lock folder and npm install again before the error cleared.

**Comment by efosdan** [comment]
completed now. kindly check so i can merge and continue

**Review by courage173 — APPROVED** [review]
Looks good!

**Pull request merged**